src/js/controller/controller.js

```diff
@@ -259,8 +259,8 @@ Object.assign(Controller.prototype, {
             });
 
 
-            // Only attempt to preload if this is the first player on the page or viewable
-            if (instances[0] === _api || viewable === 1) {
+            // Only attempt to preload an idle player if it's the first player on the page or viewable
+            if (model.get('state') === 'idle' && (instances[0] === _api || viewable === 1)) {
                 model.preloadVideo();
```

Comment:
I prefer we make this check in the `model.preloadVideo()` so all conditions outside of the api instances are in one place.